tests/integration/test_fim/test_files/test_skip/data/wazuh_conf.yaml

```diff
@@ -27,3 +27,5 @@
         value: "/dev"
     - skip_dev:
         value: SKIP
+    - frequency:
+        value: 3
```

Comment:
Maybe it is better to use at least 5 as the frequency value

Reply:
Done in https://github.com/wazuh/wazuh-qa/pull/4183/commits/54a29a023f15825acac1a805e598cf039a5fcdcf